scripts/troubleshoot/README.md

```diff
@@ -70,14 +70,14 @@ Steps:
 - Collect Subscription ID, Resource group name of the Aks-Engine Kubernetes cluster
 - Use the following command to run the script : `.\TroubleshootError_AcsEngine.ps1 -SubscriptionId <subId> -ResourceGroupName <rgName>`.
 This script will generate a TroubleshootDump.txt which collects detailed information about container health onboarding.
-Please send this file to [AskCoin](mailto:askcoin@microsoft.com). We will respond back to you.
+Please create a support ticket in Azure and send this file alongwith it. We will respond back to you.
 - Please remember to 'Set-ExecutionPolicy' to what it was previously(from the value stored in the file) after you've run the script
 
 For more details on Azure Resource Manager template deployment via cli refer to [this documentation](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-deploy-cli).
 
 If steps above did not help to resolve your issue, you can use either of the following methods to contact us for help:
 *	File a [GitHub Issue](https://github.com/microsoft/Docker-Provider/issues)
-*	Email [AskCoin](mailto:askcoin@microsoft.com) : Please attach the TroubleshootErrorDump.txt in the email generated by the troubleshooting script if you had tried running the script to solve your problem.
+*	Create a support ticket in Azure : Please attach the TroubleshootErrorDump.txt generated by the troubleshooting script if you had tried running the script to solve your problem.
```

Comment:
probable adding link for the Azure support ticket would be useful for customers. Here is the link for the support ticket - https://azure.microsoft.com/en-us/support/create-ticket